src/common/BaseManager.ts

```diff
@@ -22,9 +24,12 @@ export interface Manager {
 export default class BaseManager implements Manager {
     location: number;
 
+    fileType?: string;
+
     reactEl: HTMLElement;
 
-    constructor({ location = 1, referenceEl }: Options) {
+    constructor({ fileType, location = 1, referenceEl }: Options) {
+        this.fileType = fileType;
```

Comment:
Managers should not be aware what type of file they are being used on, as it will inevitably lead to them being coupled to specific viewer implementations.

Reply:
Sounds good, thanks for the info.